src/view/com/feeds/ProfileFeedgens.tsx

```diff
@@ -63,27 +63,27 @@ export const ProfileFeedgens = React.forwardRef<
     error,
     refetch,
   } = useProfileFeedgensQuery(did, opts)
-  const isEmpty = !isFetching && !data?.pages[0]?.feeds.length
   const {data: preferences} = usePreferencesQuery()
 
   const items = React.useMemo(() => {
+    const hasListData = data?.pages[0]?.feeds.length
     let items: any[] = []
-    if (isError && isEmpty) {
+    if (isError && !isFetching && !hasListData) {
       items = items.concat([ERROR_ITEM])
     }
     if (!isFetched && isFetching) {
       items = items.concat([LOADING])
-    } else if (isEmpty) {
+    } else if (isFetched && !hasListData) {
       items = items.concat([EMPTY])
     } else if (data?.pages) {
       for (const page of data?.pages) {
         items = items.concat(page.feeds)
       }
-    } else if (isError && !isEmpty) {
+    } else if (isError && !isFetching && hasListData) {
```

Comment:
The way this original logic was written, `LOAD_MORE_ERROR_ITEM` would be added whenever `isError` + `isFetching` are both true -- that did not seem like intentional behavior to me and seems like it could cause some weird bugs, for example if `isError` remained true while attempting refetch from `ErrorMessage`

Can revert this change if I'm missing something